perfkitbenchmarker/linux_packages/cassandra.py

```diff
@@ -164,6 +214,30 @@ def _StartCassandraIfNotRunning(vm):
     Start(vm)
 
 
+def GetCassandraCliPath(vm):
+  if vm.OS_TYPE == os_types.JUJU:
+    """
```

Comment:
There are a few places in this file where you use multi line strings to add comments - could you please switch to just using comments to stay consistent with the style guide?